creator-node/src/services/sync/UserSyncFailureCountService.ts

```diff
@@ -0,0 +1,20 @@
+const redis = require('../../redis')
+
+const HSetKey = 'UserSyncFailureCounts'
+
+/**
+ * Tracks user SyncRequest failure counts in Redis
+ */
+module.exports = class UserSyncFailureCountService {
+  static async resetFailureCount(wallet: string) {
+    await redis.hSet(HSetKey, wallet, 0)
```

Comment:
just want to confirm this is case insensitive? i've always seen it as hset/hget in the docs eg https://github.com/luin/ioredis#transforming-arguments--replies

Reply:
yep i was looking at docs for the wrong package lol (https://github.com/redis/node-redis), i know this doesn't work